packages/headless/.eslintrc.js

```diff
@@ -1,24 +1,26 @@
 module.exports = {
-  parser: '@typescript-eslint/parser',
-  parserOptions: {
-    project: 'tsconfig.json',
-    sourceType: 'module',
-  },
-  plugins: ['@typescript-eslint/eslint-plugin'],
-  extends: [
-    'plugin:@typescript-eslint/recommended',
-    'plugin:prettier/recommended',
-  ],
-  root: true,
-  env: {
-    node: true,
-    jest: true,
-  },
-  ignorePatterns: ['.eslintrc.js'],
-  rules: {
-    '@typescript-eslint/interface-name-prefix': 'off',
-    '@typescript-eslint/explicit-function-return-type': 'off',
-    '@typescript-eslint/explicit-module-boundary-types': 'off',
-    '@typescript-eslint/no-explicit-any': 'off',
-  },
+	parser: '@typescript-eslint/parser',
+	parserOptions: { 
+		project: 'tsconfig.json',
+    tsconfigRootDir: __dirname,
+		sourceType: 'module',
+	},
+	plugins: ['@typescript-eslint/eslint-plugin'],
+	extends: [
+		'plugin:@typescript-eslint/recommended',
+		'plugin:prettier/recommended', 
+	],
+	root: true,
+	env: {
+		node: true,
+		jest: true,
+	},
+	ignorePatterns: ['.eslintrc.js'],
```

Comment:
don't change linting file without our concern. if anything should be changed then let us know. but don't do alone. because if each developer tries to add something in linting files then the project linting might be messed up

Reply:
reverted.